pkg/cluster/volumes.go

```diff
@@ -451,11 +454,6 @@ func quantityToGigabyte(q resource.Quantity) int64 {
 }
 
 func (c *Cluster) executeEBSMigration() error {
-	if !c.OpConfig.EnableEBSGp3Migration {
-		return nil
-	}
-	c.logger.Infof("starting EBS gp2 to gp3 migration")
-
```

Comment:
`c.OpConfig.EnableEBSGp3Migration` is already checked before calling `executeEBSMigration`. The log line is quite confusing because the function is executed every sync and at this point it's not decided yet, that a migration is actually happening.